### DISCUSSION TITLE
New rules regarding the 280 characters limit

### DISCUSSION THREAD

**Discussion by homethreads** [danieldevine/bird-elephant#43] [General]
The Twitter API has been changed recently, to support new rules regarding the 280 characters limit.

There is a new tweet_mode (full_text) to remove this limitation, do you have this feature on in your script?

**Comment by danieldevine** [comment]
Which changes are these and which limitations are you referring to? It would help if you'd clarify which endpoint you're trying to use. As far as I'm aware tweet_mode-extended is a Twitter API v1.1 concept. Bird Elephant is for Twitter API v2.
